src/content/docs/accounts/accounts-billing/new-relic-one-pricing-billing/user-count-billing.mdx

```diff
@@ -19,23 +19,23 @@ For how to view your user count in the UI, see [Billing-related UI](/docs/accoun
 
 We have two versions of the New Relic One pricing model: 
 
-* **Our primary New Relic One pricing version.** This version is also known as "New Relic One with core users." It was released January 12, 2022. Organizations on this version have access to a third user type: the [core user](/docs/accounts/accounts-billing/new-relic-one-user-management/user-type), and their [billable user rules](#user-count) are different than the "New Relic One without core users" version's rules.
-* **The "New Relic One without core users" version.** If your New Relic organization existed prior to January 12, 2022, and hasn't switched to the newer version, this is your version. Organizations on this version can add only basic users and full platform users; they don't have access to core users. This pricing version will be increasingly deprecated over time. See the [billable user rules for this version](#non-core).
+* **Our primary New Relic One pricing version.** This version is also known as "New Relic One with core users." It was released January 12, 2022. Organizations on this version have access to a third user type: the [core user](/docs/accounts/accounts-billing/new-relic-one-user-management/user-type). And this version has different [billing calculations](#user-count) and [user downgrade rules](#user-downgrade-rules). 
+* **The "New Relic One without core users" version.** If your New Relic organization existed prior to January 12, 2022, and hasn't switched to the newer version, this is your version. Organizations on this version can add only basic users and full platform users; they don't have access to core users. This pricing version will be increasingly deprecated over time. See the [rules for this version](#non-core).
```

Comment:
If there's a date, even approximately, when this is going to be deprecated, it might be handy to put it here.